src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java

```diff
@@ -132,28 +111,14 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
                                                 //kit requests from gen2 can be removed after all studies are migrated
                                                 DDPParticipant participant = DDPParticipant.getDDPParticipant(latestKit.getBaseURL(), latestKit.getInstanceName(), kitDetail.getParticipantId(), latestKit.isHasAuth0Token());
                                                 if (participant != null) {
-                                                    // if the kit type has sub kits > like for promise
+                                                    // if the kit type has sub kits (primarily gbf)
                                                     String collaboratorParticipantId = KitRequestShipping.getCollaboratorParticipantId(latestKit.getBaseURL(), latestKit.getInstanceID(), latestKit.isMigrated(),
                                                             latestKit.getCollaboratorIdPrefix(), participant.getParticipantId(), participant.getShortId(), kitRequestSettings.getCollaboratorParticipantLengthOverwrite());
-                                                    //only testboston for now which is not gen2 so it won't matter
-                                                    if (kitHasSubKits) {
-                                                        List<KitSubKits> subKits = kitRequestSettings.getSubKits();
-                                                        String externalOrderNumber = addSubKits(subKits, kitDetail, collaboratorParticipantId, kitRequestSettings, latestKit.getInstanceID(), null);
```

Comment:
I think you removed too much, I don't see subkits getting written into db anymore